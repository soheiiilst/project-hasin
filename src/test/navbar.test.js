import React from 'react';
import { shallow } from 'enzyme';
import Navbar from '../component/navbar/navbar.component';

it('expect to render Navbar component', () => {
  expect(shallow(<Navbar />)).toMatchSnapshot();
})
