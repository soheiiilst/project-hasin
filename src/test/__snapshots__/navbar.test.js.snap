// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect to render Navbar component 1`] = `ShallowWrapper {}`;
